sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -772,6 +772,7 @@ public async Task VerifyUpdateCertificatePolicy()
         [RecordedTest]
         [TestCase("application/x-pkcs12")]
         [TestCase("application/x-pem-file")]
+        [LiveOnly(Reason = "Certificate data cannot be stored in test recording.")]
```

Comment:
I'm not crazy about this messaging. It will need to be. Instead, a tracking issue should be opened and the reason can point to that as something we need to track and eventually re-enable these tests.

All key and secret material herein is created upon test execution and has no associated with accounts. This would tear a huge hole in testing coverage since Key Vault is all about secrets, keys, and certificates.

Reply:
These don't need to be marked LiveOnly. I've removed the attribute.